deploy/before_migrate.rb

```diff
@@ -0,0 +1,9 @@
+rails_env = new_resource.environment["RAILS_ENV"]
+secret_key = new_resource.environment["SECRET_KEY_BASE"]
+Chef::Log.info("Precompiling assets for RAILS_ENV=#{rails_env}...")
```

Comment:
This is going to fail if Chef is not in place. Can't we simply use Rails' logger?